setup.py

```diff
@@ -46,7 +46,6 @@
 install_requires = [
     'Flask-SQLAlchemy>=2.2',
     'Flask>=0.12',
-    'Flask-Login<0.5.0',  # FIXME waiting for Flask-Security release
```

Comment:
Is removing `Flask-Login` pin safe here? (no invenio in r-w-controller)

Reply:
It was my bad, I shouldn't have pinned in the first place since there are no `Flask-Login` nor `Flask-Security` in RWC 😅.